"use server";

export const onFollow = async (id: string) => {
  try {
  } catch (err) {
    throw new Error("Internal error");
  }
};
